### PR TITLE
feat: update index and beta pages

### PR DESCRIPTION
**Pull request by cfu288**
Opening a draft PR for proposed changes as part of the re-organization of the docs repo.

I think the first three pages to the docs site should be the following:
1) Welcome - Intro, explain what Fasten is, give user actionable next steps
2) What can Fasten do? - List and demo of current features
3) Getting started with Fasten - Installation instructions for the Fasten beta

Current tasks:
- [x] Improve welcome/index page with gentler onboarding and actionable buttons to guide users to next steps
- [ ] Add a "What can fasten do?" page with a quick overview of Fasten in its current state with videos/pictures.
- [x] Update the beta page to a "Getting started" page with guided installation instructions

### CODE REVIEW

File: BETA.md [hunk 1]
```diff
@@ -1,111 +1,167 @@
...
+| Cerner                | Username: `nancysmart` <br>Password: `Cerner01`                            | [test accounts](https://docs.google.com/document/d/10RnVyF1etl_17pyCyK96tyhUWRbrTyEcqpwzW-Z-Ybs/edit)                                 |
+| Cigna                 | Username: `syntheticuser05` <br>Password: `5ynthU5er5`                     | [test accounts](https://developer.cigna.com/service-apis/patient-access/sandbox#How-to-Use-the-Sandbox-Sandbox-Test-Users)            |
+| eClinicalWorks/Healow | Username: `AdultFemaleFHIR` <br>Password: `e@CWFHIR1`                      | [test accounts](https://fhir.eclinicalworks.com/ecwopendev/)                                                                          |
+| Epic                  | Username: `fhircamila` <br>Password: `epicepic1`                           | [test accounts](https://fhir.epic.com/Documentation?docId=testpatients)                                                               |
+| HealthIT              | Username: `demouser` <br>Password: `Demouser1!`                            | [test accounts](https://fhirsandbox.healthit.gov/secure/r4/view/userlogin.html)                                                       |
```
Comment (AnalogJ) [inline BETA.md:143]: did you intend to keep this credentials table in the doc 2 times?

**Comment by AnalogJ** [comment]
This is a fantastic start, thanks @cfu288 !

File: BETA.md (hunk 1 above)
Comment (cfu288) [reply BETA.md:124]: Whoops, obv not

File: beta/main.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+layout: default
+title: Setting up Fasten to Use With Real Medical Data
+parent: Getting started
+description: "Setting up Fasten to use with real medical data"
+nav_order: 1
+---
+
+# Setting up Fasten to Use With Real Medical Data
+
+This version allows you to connect to 650+ different Healthcare providers, using your existing accounts. It will allow you to connect and retrive your full electronic medical record and store it within Fasten.
```
Comment (AnalogJ) [inline beta/main.md:11]: 27,000+ institutions

**Comment by AnalogJ** [comment]
Hey @cfu288 this looks pretty good, other than a couple of small nits. However its still in Draft, should I mark it ready for review and merge it? Or do you have plans to make more changes?

**Comment by cfu288** [comment]
Ah, in my mind its a WIP but if you're happy with it we can merge it now and I'll make another PR with the "What can fasten do?" feature demo page.

**Comment by AnalogJ** [comment]
yeah, I lean towards making small PR's without long lived branches. I plan on making some changes to the docs repo myself as well, and I don't want to introduce any conflicts with your work.

**Comment by cfu288** [comment]
Fair enough, I'm done with changes for now - just pushed some last minute typo fixes

**Comment by AnalogJ** [comment]
awesome, I merged your changes and they should be visible on docs.fastenhealth.com in a couple minutes.

I'll also add some info to the pre-v0.1.9 release notes issue

**Pull request merged**